### PR TITLE
Add OWNERS files

### PR DESCRIPTION
**Pull request by alvaroaleman**
**What this PR does / why we need it**:

Adds owners file so Prow will know who has to approve what

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: api/pkg/resources/metrics-server/OWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - team-seed
```
Comment (mrIncompetent) [inline api/pkg/resources/metrics-server/OWNERS]: I would put this rather into `team-lifecycle`.
It has actually nothing to do with monitoring of the cluster. Its a required extension API server to enable `kubectl top {node|pod}` & used by the scheduler

**Comment by mrIncompetent** [comment]
The cmd directory could use some OWNERS files as well

**Comment by mrIncompetent** [comment]
The config directory as well

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    - thz
+
+  team-seed:
+    - kron4eg
+    - xrstf
+    - thetechnick
+    - xmudrii
+    - zreigz
+
+  team-ui:
```
Comment (p0lyn0mial) [inline OWNERS_ALIASES:15]: please add `zreigz` and `floreks`

**Comment by alvaroaleman** [comment]
> the cmd directory could use some OWNERS files as well

It does have them

> The config directory as well

I disagree because the things there are owned by all teams. Only thing I could do is add OWNERS files for e.g. the monitoring chart

**Comment by p0lyn0mial** [comment]
package `https://github.com/kubermatic/kubermatic/tree/master/api/pkg/handler` belongs to `team-ui`

**Comment by alvaroaleman** [comment]
@p0lyn0mial Yeah, I configured that, check out the `api/pkg/handler/OWNERS` file

**Comment by p0lyn0mial** [comment]
please add `https://github.com/kubermatic/kubermatic/tree/master/api/cmd/rbac-generator` and `https://github.com/kubermatic/kubermatic/tree/master/api/cmd/projects-migrator` to `team-ui`

**Comment by alvaroaleman** [comment]
@mrIncompetent @p0lyn0mial updated, PTAL

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    - kdomanski
+    - nikhita
+    - thz
+
+  team-seed:
+    - kron4eg
+    - xrstf
+    - thetechnick
+    - xmudrii
+    - zreigz
```
Comment (p0lyn0mial) [inline OWNERS_ALIASES]: I think that `zreigz` is a member of `team-ui` :)

**Comment by xrstf** [comment]
>> The config directory as well
>
> I disagree because the things there are owned by all teams. Only thing I could do is add OWNERS files for e.g. the monitoring chart

Sounds sensible to make `config/monitoring`, `config/logging` and `config/backup` owned by the seed-team.

**Review by kron4eg — DISMISSED** [review]

**Review by mrIncompetent — DISMISSED** [review]

**Comment by alvaroaleman** [comment]
@mrIncompetent @p0lyn0mial Adressed everything, PTAL

**Review by mrIncompetent — APPROVED** [review]

**Review by p0lyn0mial — APPROVED** [review]

**Pull request merged**